src/test-e2e/login/login-e2e.spec.ts

```diff
@@ -16,7 +16,7 @@ describe('Login', () => {
 
   it('- should reach log in page', () => {
     expect(loginPage.isLoginPage()).toBeTruthy();
-    expect<any>(loginPage.getTitle()).toEqual('STRATOS');
+    expect<any>(loginPage.getTitle()).toEqual('');
```

Comment:
Can we remove this altogether - it won't work on forks, if they unhide the title.